API/controlsClass.m

```diff
@@ -33,6 +33,8 @@
     %     Which procedure RAT should execute. Can be 'calculate', 'simplex', 'de', 'ns', or 'dream'.
     % calcSldDuringFit : logical, default: false
     %     Whether SLD will be calculated during fit (for live plotting etc.)
+    % simulationNPoints : whole number, default: 500
```

Comment:
why not `numSimulationPoints` or `nSimulationPoints` like the other parameters?

Reply:
I was looking at `resampleNPoints` which is also a universal parameter. But let's go for `numSimulationPoints` instead.